core/controllers/reader.py

```diff
@@ -300,7 +300,37 @@ def get(self, exploration_id):
 
 
 class StorePlaythroughHandler(base.BaseHandler):
-    """Handles a useful playthrough coming in from the frontend to store it."""
+    """Handles a useful playthrough coming in from the frontend to store it. If
+    the playthrough already exists, it is updated in the datastore.
+    """
+
+    def _is_issue_in_exp_issues_model(
+            self, exp_issues, issue_type, customization_args):
+        """Checks whether the passed in issue exists in the unresolved issues
+        list of the exploration issues model.
+
+        Args:
+            exp_issues: ExplorationIssues. The exploration issues domain object.
+            issue_type: str. The type of the issue obtained from the playthrough
+                data.
+            customization_args: dict. The customization args dict
```

Comment:
Document as: `dict(str : *)`, add period to end.

Reply:
Done.